.gitattributes

```diff
@@ -0,0 +1,4 @@
+/.git*          export-ignore
+/composer.json  export-ignore
```

Comment:
Maybe don't ignore this one?

Reply:
Why not? What can be the purpose of distributing composer.json?

Reply:
Gentle reminder